### PR TITLE
added login and signup component with chakra un

### PR DESCRIPTION
**Pull request by joybiswas704**
Hi i added the login and signup components. Please review the code

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
```
Comment (saifulislamdev) [inline src/components/App.js:9]: Not reviewing the PR, but interested to see how you did certain things. One thing I noticed is that the spacing is huge across files. Can we set the tab size to 2 spaces?

Comment (joybiswas704) [reply src/components/App.js:9]: can you be more specific regarding the spacing? Is it spacing inside the source code or the layout of the actual components?

Comment (saifulislamdev) [reply src/components/App.js:9]: Spacing inside the source.
```html
<ChakraProvider>
	<Container maxWidth='container.xl' p={0}>
		<Router>
			<Switch>
				<Route path='/' exact>
					<SignUp />
				</Route>
				<Route path='/login' exact>
					<LogIn />
				</Route>
			</Switch>
		</Router>
	</Container>
</ChakraProvider>
```

can be condensed to 

```html
<ChakraProvider>
  <Container maxWidth='container.xl' p={0}>
    <Router>
      <Switch>
        <Route path='/' exact>
          <SignUp />
        </Route>
        <Route path='/login' exact>
          <LogIn />
        </Route>
      </Switch>
    </Router>
  </Container>
</ChakraProvider>
```

Comment (saifulislamdev) [reply src/components/App.js:9]: I use prettier as well, but I am still able to set the tab size to 2 spaces.

File: src/components/App.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+		<ChakraProvider>
+			<Container maxWidth='container.xl' p={0}>
+				<Router>
+					<Switch>
+						<Route path='/' exact>
```
Comment (zealptl) [inline src/components/App.js:14]: Can you move signup page to '/auth/signup'. Want to keep '/' for main page

File: src/components/LogIn/LogIn.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	return (
+		<div>
+			<Flex h='100vh' py={20}>
+				<VStack
+					w='full'
+					h='full'
+					p={10}
+					spacing={10}
+					alignItems='flex-start'
+					bg='#E2E8F0'
```
Comment (zealptl) [inline src/components/LogIn/LogIn.js:60]: can you change bg to 'gray.50'

File: src/components/LogIn/LogIn.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+	const handleClick = async (e) => {
+		e.preventDefault();
+		setError(false); // resetting the error messgae
+		const isValid = await logInSchema.isValid({
+			email: email,
+			password: password,
+		});
+
+		if (isValid) {
+			const URI = 'http://localhost:5000/login'; // URI is subject to change
```
Comment (zealptl) [inline src/components/LogIn/LogIn.js:41]: api path will be 'http://localhost:5000/api/auth/signin' on backend

File: src/components/SignUp/SignUp.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+		const isValid = await signupSchema.isValid({
+			firstName: firstName,
+			lastName: lastName,
+			email: email,
+			password: password,
+			repeatPassword: repeatPassword,
+		});
+
+		if (isValid) {
+			const URI = 'http://localhost:5000/signup'; // URI is subject to change
```
Comment (zealptl) [inline src/components/SignUp/SignUp.js:51]: api path will be 'http://localhost:5000/api/auth/signup' on backend

File: src/components/SignUp/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+	return (
+		<Flex h='100vh' py={20}>
+			<VStack
+				w='full'
+				h='full'
+				p={10}
+				spacing={10}
+				alignItems='flex-start'
+				bg='#E2E8F0'
```
Comment (zealptl) [inline src/components/SignUp/SignUp.js:74]: can you change bg to 'gray.50'

File: src/components/App.js (hunk 1 above)
Comment (joybiswas704) [reply src/components/App.js:9]: I agree to @zealptl . I think its the github cause in my source code the spacing is minimal

File: src/assets/login.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...l25.35068-.74561C391.76421,275.08,394.16732,292.48081,399.1716,307.90158Z" transform="translate(-203.5 -174.13424)" fill="#ffb8b8"/><path d="M409.41752,423.55243c-27.13873,18.49308-46.31418.63272-60.94729-26.92346,2.03338-16.86188-1.259-37.04061-7.35672-58.96635a40.13762,40.13762,0,0,1,24.50567-48.40124h0l32.06116,13.421c27.22362,22.19038,32.582,46.227,22.36825,71.5784Z" transform="translate(-203.5 -174.13424)" fill="#e2e8f0"/><path d="M331.32124,326.54178,301.4969,342.19956l52.9382,31.31555,7.366,18.16951a9.63673,9.63673,0,0,1-5.78925,12.73088h0a9.63673,9.63673,0,0,1-12.76159-8.54442l-.74489-12.66307-67.2838-22.20366a15.73306,15.73306,0,0,1-9.87265-9.61147v0a15.733,15.733,0,0,1,5.90262-18.30258l54.10485-37.11845Z" transform="translate(-203.5 -174.13424)" fill="#ffb8b8"/><path d="M361.14557,329.52422c-12.43861-5.4511-23.74934.47044-38.026,5.21926l-2.23683-39.51725c14.17612-7.55568,27.69209-9.59281,40.26285-3.728Z" transform="translate(-203.5 -174.13424)" fill="#e2e8f0"/><circle cx="172.52496" cy="78.09251" r="23.80211" fill="#ffb8b8"/><path d="M404.5,249.22353c-23.56616,2.30811-41.52338-1.54606-53-12.52007v-8.8377h51Z" transform="translate(-203.5 -174.13424)" fill="#2f2e41"/></svg>
```
[the remark this answers is not in the document]
Comment (joybiswas704) [reply src/assets/login.svg:1]: @zealptl sure. I didnt know i can save the svg from figma.

File: src/components/App.js (hunk 2 above)
Comment (joybiswas704) [reply src/components/App.js:14]: @zealptl sure thing!

File: src/components/SignUp/SignUp.js (hunk 6 above)
Comment (joybiswas704) [reply src/components/SignUp/SignUp.js:74]: done

File: src/components/SignUp/SignUp.js (hunk 5 above)
Comment (joybiswas704) [reply src/components/SignUp/SignUp.js:51]: done

File: src/components/LogIn/LogIn.js (hunk 4 above)
Comment (joybiswas704) [reply src/components/LogIn/LogIn.js:41]: done

File: src/components/App.js [hunk 8]
```diff
@@ -1,25 +1,27 @@
...
+							<SignUp />
+						</Route>
+						<Route path='/login' exact>
```
[the remark this answers is not in the document]
Comment (joybiswas704) [reply src/components/App.js]: done

**Review by zealptl — APPROVED** [review]

**Pull request merged**